log_handler.py

```diff
@@ -134,6 +134,9 @@ def emit(self, record):
 
 class StatusBarHandler(logging.Handler):
     def emit(self, record):
+        if record.levelno != logging.WARNING:
```

Comment:
Shouldn’t you use `STATUS_BAR_LEVEL` here?

Reply:
That would be clever indeed.

Reply:
Do we want to filter everything that's not the specified error or just `exception` output? That uses the ERROR level as well, so the only way to filter would be to check for a log record's exc_info attribute.

Reply:
The intention here was to only show warnings in the status bar, for errors we already open the error panel.